package dmitriy.losev.network.exception

const val NETWORK_CALL_FAIL = -2

const val BAD_REQUEST_EXCEPTION_CODE = 400

const val INTERNAL_SERVER_ERROR = 500

const val NO_INTERNET_EXCEPTION_CODE = 666

const val NOT_FOUND = 404